README.md

```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
+At this stage, `Colourblind8` will focus on developing settings for the three most prevalent colourblind perspectives - protanopia, deuteranopia and tritanopia (see [Appendix](https://github.com/UBC-MDS/Colourblind8#Appendix) for more information on the types of colourblindness).
 
 
-### Function Discription
+### Function Description
+
+
+In this package, three functions are included to make `matplotlib`plots more interpretable for people with protanopia, deuteranopia and tritanopia. Within each function, users can select one of our customized colour palettes via the `palette` argument according to their preference: `deutera` for deuteranopia,`prota` for protanopia ,`trita`for tritanopia. The following are our main functions:
 
 - `plot_lines()`:
 
-  - This function implements a method that makes line graphs accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, transparency of the objects, legend and axes size as well as the face colour.
+  - This function implements a method that enhances line graphs through modifying the colour of lines, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.
 
    **inputs:** x variable,  y variable (a list of lists), transparency, labels of each y variable, palette colour, title, x labels, y labels and legend title.    
-   **output:** a matplotlib object  
+   **output:** a matplotlib line graph  
 
 
 - `plot_scatter()`:
 
-  - This function implements a method that makes scatterplots accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size, transparancy of the objects, as well as the face colour.
+  - This function implements a method that enhances scatterplots through modifying the colour of points, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.
 
    **inputs:** x variable,  y variable (a list of lists), transparency, labels of each y variable, palette colour, title, x labels, y labels and legend title.    
-   **output:** a matplotlib object.  
+   **output:** a matplotlib scatterplot.
+
 
+- `plot_histogram()`:
 
-.- `plot_histogram()`:
+   - This functions implements a method that enhances histograms through modifying the colour of histogram layers, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.
 
-   - This functions implements a method that makes histograms accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size, transparancy of the objects number of bins,  as well as the face colour.
```

Comment:
This function implement a method that creates histograms by initializing the colour of histogram layers, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes

Reply:
change made to new Readme.